.changeset/sweet-fishes-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-pull-requests-board': patch
+---
+
+Add optional `defaultLimit` prop to `EntityTeamPullRequestsCard` and `EntityTeamPullRequestsContent` to limit the number of PRs shown per repository. Excluding this prop will result in showing all PRs for each repository.
```

Comment:
I worry a little that this would be unexpected for current users that they would see all of their PRs once they upgrade. Could this possibly cause performance issues? It would be great to keep the current state and make this additive

Reply:
Good point, I changed the logic around to default to the original functionality.